.travis.yml

```diff
@@ -9,10 +16,10 @@ stages:
   - test
   - name: deploy
     if: type != pull_request
+script: make verify build build-integration build-e2e test images svcat
 jobs:
   include:
-    # Test
-    - script: make verify build build-integration build-e2e test images svcat
+    # Test is implicit from the build matrix
     # Deploy
     - stage: deploy
```

Comment:
@MHBauer Do we need to add `script: skip` here? I would think the deploy stage would also run the script specified above otherwise.

Reply:
script is defined and overridden below, right? 
`         script: contrib/travis/deploy.sh`?

Reply:
Ah, whoops; my mistake, you're right. :smiley: